From f498c31816af4d47800185b0f4036c89a5c522af Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
Date: Sun, 28 Jun 2020 19:44:18 +0200
Subject: [PATCH 29/51] cht-bsw-rt5672: Add Lenovo Miix 2 10 specific
 configuration

By default the cht-bsw-rt5672 config lets the user figure out if a device
has a Mono spaker or Stereo speakers and which DMIC input is used.

Since this is not very user-friendly we do regex matches on the DMI derived
longname to automaticaly select the right options (and to only advertise
those options).

This commit adds support for automatically selecting the right options
on the Lenovo Miix 2 10 tablet. Since this is the first device for which
we do longname matching using a DMIC on DMIC1, this also fixes a fixme :)

Signed-off-by: Hans de Goede <hdegoede@redhat.com>
Signed-off-by: Jaroslav Kysela <perex@perex.cz>
---
 ucm2/cht-bsw-rt5672/HiFi.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ucm2/cht-bsw-rt5672/HiFi.conf b/ucm2/cht-bsw-rt5672/HiFi.conf
index dc8ddd5ed45f..94de7dbb1504 100644
--- a/ucm2/cht-bsw-rt5672/HiFi.conf
+++ b/ucm2/cht-bsw-rt5672/HiFi.conf
@@ -11,7 +11,7 @@ If.cfg-dmic1 {
 	Condition {
 		Type RegexMatch
 		String "${CardLongName}"
-		Regex "(SoMeThInK1)" # fixme!
+		Regex "(LENOVO.*LenovoMiix210)"
 	}
 	True {
 		Define.MonoSpeaker ""
-- 
2.16.4

